docs-src/styles/tables.scss

```
@@ -1,36 +1,32 @@
-// From Bootstrap: http://getbootstrap.com/customize/?id=e516e5d6e28288745392
-table {
+caption {
```

We can prefix this with the `table` class as well, and probably move it to after the `.table` block.

```suggestion
.table caption {
```

---

Sure!

---

Done!